packages/langium/src/grammar/language-meta-data.ts

```diff
@@ -0,0 +1,10 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+export interface LanguageMetaData {
+    languageId: string;
+    extensions: string[];
```

Comment:
Shall we name this property `fileExtensions` for clarity? "extensions" is quite generic.